From 774bed086468190ba47299fb58d9aac863a6de4d Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Tue, 28 Aug 2012 12:15:06 +0300
Subject: [PATCH 330/370] conf: declare c2_conf_reg_tl in conf/obj_ops.h

List descriptor (c2_conf_reg_tl) and list operations will be defined
in conf/reg.c.  We declare them in "private" header so that they are
hidden from the application.

RB: r/981/diff/2-3/?file=27909#file27909line78
---
 conf/obj_ops.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/conf/obj_ops.h b/conf/obj_ops.h
index cbdf160..5c4a534 100644
--- a/conf/obj_ops.h
+++ b/conf/obj_ops.h
@@ -59,6 +59,11 @@
 /** Magic value of a generic configuration object (c2_conf_obj). */
 enum { C2_CONF_OBJ_MAGIC = 0x335e1f1e55ce1177 /* selfless cell */ };
 
+/* export */
+C2_TL_DESCR_DECLARE(c2_conf_reg, extern);
+C2_TL_DECLARE(c2_conf_reg, , struct c2_conf_obj);
+
+/* import */
 struct c2_conf_reg;
 
 /** Configuration object operations. */
-- 
1.8.3.2

